_pages/en_US/installing-unlaunch.md

```diff
@@ -14,7 +14,7 @@ Unlaunch is an exploit that takes place on system boot. This allows it to have h
 - Better sound in GBARunner2
 - Brick-protection
 
-If you have not yet done so, please made a [NAND backup](dumping-nand). While the chances are slim, Unlaunch can accidentally brick your Nintendo DSi. A NAND backup + [hardmod](https://web.archive.org/web/20151102221503/https://gbatemp.net/threads/dsi-downgrading-the-complete-guide.393682/) would allow you to restore this backup, provided you know how to solder.
+If you have not yet done so, please made a [NAND backup](dumping-nand). While the chances are slim, Unlaunch can accidentally brick your Nintendo DSi. A NAND backup + [hardmod](https://wiki.ds-homebrew.com/ds-index/hardmod) would allow you to restore this backup, provided you know how to solder.
```

Comment:
You could just say `follow [Dumping NAND]`. 

What's already there is a typo lol